common/templates/common/blocks/aligned_captioned_image.html

```diff
@@ -1,6 +1,14 @@
 <figure class="inline-media {{ value.alignment }}">
     {% include 'common/blocks/image_block.html' with value=value.image %}
-    {% if value.caption %}
-        <figcaption class="inline-media__caption">{{ value.caption }}</figcaption>
+    {% if value.caption or value.image.attribution %}
+        <figcaption class="inline-media__caption">
+			{% if value.caption %}{{ value.caption }}{% endif %}
+			{% if value.image.attribution %}
+				<span
+					class="media-attribution"
+					aria-label="Attributed to: {{ value.image.attribution }}"> — {{ value.image.attribution }}
```

Comment:
I don't think an `aria-label` makes sense in a span, since it is not interactive and many screen readers ignore it because it has no interactive role. I feel the figcaption giving the accessible name to the image will also ignore the aria-label part. If we really want to mention "Attributed to: " then we can place that text inside another span with `.sr-only`